.storybook/preview-head.html

```diff
@@ -0,0 +1,3 @@
+<script>
+  document.querySelector('html').setAttribute('dir', 'rtl');
```

Comment:
The nub of the question is how to parametrise the arguments currently set as `dir`, and `rtl` in a way that allows them to be set per story variant.